app/components/ui/ColorSwitch.vue

```
@@ -1,30 +1,32 @@
-    <Icon name="line-md:moon-filled-alt-to-sunny-filled-loop-transition" />
-    <Icon name="line-md:moon-filled-alt-loop" />
-  </div>
-  <button @click="switchColor" class="bg-transparent border-none flex inline-block cursor-pointer mr-0"
-    aria-label="Color Switcher">
-    <Icon :name="colorIcon" class="h-6 w-6 transition-all duration-500 dark:text-white text-black" />
-  </button>
-</template>
-
 <script lang="ts" setup>
-const colorMode = useColorMode();
+const colorMode = useColorMode()
```

ESLint

[error] 'useColorMode' is not defined.

(no-undef)